frontend/src/app/dynamic-form/form-submission-dialog.html

```diff
@@ -0,0 +1,4 @@
+<h1 mat-dialog-title>Observation submitted!</h1>
+<div mat-dialog-actions align="center">
+    <button mat-button (click)="onNoClick()">Ok</button>
```

Comment:
The method name is somewhat confusing here.

The name "onNoClick()" implies that it wouldn't be called when a click happens.